src/Symfony/Component/Config/Tests/Definition/Builder/ArrayNodeDefinitionTest.php

```diff
@@ -54,12 +54,23 @@ public function providePrototypeNodeSpecificCalls()
             array('defaultValue', array(array())),
             array('addDefaultChildrenIfNoneSet', array()),
             array('requiresAtLeastOneElement', array()),
-            array('cannotBeEmpty', array()),
             array('useAttributeAsKey', array('foo')),
         );
     }
 
     /**
+     * @group legacy
+     * @expectedDeprecation Calling cannotBeEmpty() has no effect, use requiresAtLeastOneElement() instead. In 4.0 both methods will act the same.
+     */
+    public function testPrototypeNodeCannotBeEmpty()
```

Comment:
basically only this + above removal needs to be merged upstream using excpectedException again.